frontend/public/components/namespace.jsx

```diff
@@ -82,11 +92,41 @@ import {
 import { removeQueryArgument } from './utils/router';
 import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
 
+import NamespaceDropDown from './namespace-dropdown';
+import { isCurrentUser, isOtherUser, isSystemNamespace } from './factory/table-filters';
+
 const getModel = (useProjects) => (useProjects ? ProjectModel : NamespaceModel);
 const getDisplayName = (obj) =>
   _.get(obj, ['metadata', 'annotations', 'openshift.io/display-name']);
+
 const CREATE_NEW_RESOURCE = '#CREATE_RESOURCE_ACTION#';
 
+export const REQUEST_FILTER = { me: 'me', user: 'user', system: 'system' };
+
+const getFilters = () => [
+  {
+    filterGroupName: 'Requester',
```

Comment:
Should this be translated?

Reply:
Yes it should.  Thanks for catching this.  Changes on next commit.